.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+orbs:
+  composer: itinerisltd/composer@0.2
+
+workflows:
+  test:
+    jobs:
+      - composer/install:
+          name: test
+          executor: composer/seven
+          pre-steps:
+            - run: sudo docker-php-ext-enable xdebug
```

Comment:
https://github.com/roots/wordpress-packager/blob/2e22147f289b5f3a5dfba51f35bc786643ca2340/build/dev/Dockerfile already has this